src/views/MediaChannels/components/Sidebar.tsx

```diff
@@ -27,21 +28,50 @@ export const Sitebar: React.FC<SitebarProps> = ({
 }) => {
   const classes = useStyles();
   const intl = useIntl();
+  const [searchText, setSearchText] = useState(criteria.name);
```

Comment:
what's this supposed to do?

Reply:
This creates a state variable, initialized with the criteria.name.

Reply:
And this is needed to debounce the change event, so not every keystroke ends up with a query to pkb

Reply:
sorry, I mean the whole commit (except fixes to footer menu)

Reply:
> this is needed to debounce the change event, so not every keystroke ends up with a query to pkb